ansible/roles/setup_database/tasks/create_db.yml

```diff
@@ -17,11 +17,10 @@
   - import/sinograms.sql
   - import/sinogram_subglyphs.sql
 
-- name: Fill languages table
-  shell: "{{code_dir}}/app/Console/cake -app {{code_dir}}/app init_languages_table"
-
```

Comment:
The migration broke this, but it’s required to get the page /stats/sentences_by_language to work. I [fixed it in the tatoeba repository](https://github.com/Tatoeba/tatoeba2/commit/d73ae0fd73946cd346b8fa09911fabaf2e8e55b1), so you can add that back now.

Reply:
Thanks for fixing that, it works now. I updated the pull request.